x/did/genesis.go

```diff
@@ -0,0 +1,34 @@
+package did
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/x/did/keeper"
+	"github.com/medibloc/panacea-core/x/did/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
+	for bz, doc := range data.Documents {
+		var key types.GenesisDIDDocumentKey
+		if err := key.Unmarshal(bz); err != nil {
+			panic(err)
+		}
+		k.SetDIDDocument(ctx, key.DID, *doc)
+	}
```

Comment:
What do you think to remove the `GenesisDIDDocumentKey` type? It has been defined previously because we had to handle the `DID` type which is not primitive (`string`).
```suggestion
	for did, doc := range data.Documents {
		k.SetDIDDocument(ctx, did, *doc)
	}
```

For now, we don't use type aliasing. So, we can just use string keys in the map as they are, IMHO. I also did it in the PR #124 .

Reply:
Also, we can check if the `did` is valid, before calling `SetDIDDocument()`.

Reply:
@youngjoon-lee 
I understood it well.
I'll fix it.